package.json

```diff
@@ -21,6 +21,7 @@
     "lint": "npm run lint:css && npm run eslint",
     "lint:css": "sgmf-scripts --lint css",
     "upload:embedded_sfra": "sgmf-scripts --uploadCartridge int_bolt_embedded_sfra",
+    "upload:sfra_ppc": "sgmf-scripts --uploadCartridge int_bolt_sfra_ppc",
     "upload:embedded_sfra_custom": "sgmf-scripts --uploadCartridge int_bolt_embedded_sfra_custom",
     "upload:all": "npm run upload:embedded_sfra && npm run upload:embedded_sfra_custom ",
```

Comment:
does "upload:all" need to include "upload:sfra_ppc"?

Reply:
no, it only includes embedded_sfra and embedded_sfra_custom,  you can check link 26 in package.json.